.github/workflows/tests.yml

```diff
@@ -23,8 +23,9 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Run Unit Tests
-        run: ./runme.sh -t
-        working-directory: ./CMR/python
+        run: |
+          pip3 install coverage
+          coverage run --source=CMR/python -m unittest discover -s CMR/python
```

Comment:
`coverage` likes to generate relative paths which confused codecov. The native codecov path fixes were ignored. Run from the root, the paths for the python code are now correct in the report, letting codecov map the coverage to the source so the reporting is now browseable inside codecov.

Reply:
Maybe we want to leave a comment detailing that in this file

Reply:
is this test something that you would not run locally? the project documents using runme.sh for common tasks, but if this is something that only gets run on github, then I can get behind it.

Reply:
got my answer, this call results in different output then what a user would get from runme.sh, this one is machine readable.